src/test/rpc/AccountTx_test.cpp

```diff
@@ -667,12 +666,8 @@ class AccountTx_test : public beast::unit_test::suite
     void
     run() override
     {
-        for (auto testVersion = RPC::apiMinimumSupportedVersion;
-             testVersion <= RPC::apiBetaVersion;
-             ++testVersion)
-        {
-            testParameters(testVersion);
-        }
+        test::jtx::forAllApiVersions(
+            [this](auto version) { testParameters(version); });
```

Comment:
You could try to use `std::bind_front` from `<functional>` here as an experiment. See what you prefer in the end. Both work of course 👍

Reply:
Thanks @godexsoft , I actually love this suggestion - makes the code so much cleaner.